docs/guides/expectations/how-to-create-and-edit-expectations-in-bulk.md

```diff
@@ -2,4 +2,96 @@
 title: ✳ How to create and edit Expectations in bulk
 ---
 
-This article is a stub.
+The `JsonSchemaProfiler` helps you quickly create [Expectation Suites](/docs/reference/expectation-suite-operations) from jsonschema files.
+
+:::note Prerequisites 
+This how-to guide assumes you have already:
+
```

Comment:
After giving this some more thought, can we convert this to the `<Prerequisites>` React component? They both look the same when rendered but Taylor added error handling and a few other bits of functionality that I'd love to have in here just for consistency.